wrapanapi/systems/rhevm.py

```diff
@@ -252,16 +252,20 @@ def _get_state(self):
     @property
     def ip(self):
         """
-        Returns IPv4 or global IPv6 address of the VM/instance
+        Returns IPv4 or global IPv6 addresses of the VM/instance
```

Comment:
**REQUEST**: Remove `instance` as RHV uses VMs, not instances.

Reply:
This was in several places. In general we aren't strict about that in wrapanapi docblocks. They're interchangeable terms for all intents and purposes.